test-requirements.txt

```diff
@@ -4,3 +4,6 @@
 
 hacking>=0.12.0,<0.13 # Apache-2.0 - test pep8
 oslotest<=3.5.0 # Apache-2.0 - test py35
+coverage>=4.5.1
+coveralls>=1.5.0
```

Comment:
I guess this lib is not necessary, because we are not using the command `coveralls` with scripts `tox`.